pkgs/development/compilers/ghc/common-hadrian.nix

```diff
@@ -477,6 +477,11 @@ stdenv.mkDerivation ({
   preInstall = ''
     pushd _build/bindist/*
 
+    # the bindist configure script uses different env variables than the GHC configure script
+    # see https://github.com/NixOS/nixpkgs/issues/267250
```

Comment:
Can you link your comment on the upstream issue with GHC as well? Or even open a new GHC issue for this? I do think this deserves some attention upstream.

Reply:
Done, opened a new issue upstream: https://gitlab.haskell.org/ghc/ghc/-/issues/24211